data/hai/hai outfits.txt

```diff
@@ -306,7 +306,7 @@ outfit "Hai Tracker Pod"
 		"die effect" "tracker impact"
 		"inaccuracy" 20
 		"velocity" 14
-		"lifetime" 600
+		"lifetime" 475
```

Comment:
```suggestion
		"lifetime" 600
		"range override" 5500
```
it takes about 3000 meters for the tracker to turn around and reach the target back in my tests